buildkit/README.md

```diff
@@ -63,7 +63,7 @@ task.tekton.dev/buildkit created
 
 ## Workspaces
 
-* **source**: A `git`-type `PipelineResource` specifying the location of the source to build.
+* **source**: A [Workspace](https://github.com/tektoncd/pipeline/blob/master/docs/workspaces.md) volume containing the source to build.
```

Comment:
total nit but I'd probably drop the word "volume" from these - not sure if a "workspace volume" is a thing? or maybe this is trying to distinguish from secrets/configmaps, etc.? if so, i feel like this might indicate that we should allow Tasks to declare more type info on a workspace to indicate what type is required 🤔  (which might be in line with your credential work?)

Reply:
Ah, I hadn't put that much thought into it.  I'll make a pr with "volume" removed.